ci-operator/step-registry/cri-o/setup/test/cri-o-setup-test-commands.sh

```diff
@@ -22,7 +22,9 @@ gcloud compute images create "crio-setup-${currentDate}" --source-image-family="
 # Delete images older than 2 weeks
 images=$(gcloud compute images list --project="openshift-node-devel" --filter="family:crio-setup AND creationTimestamp<$(date -d '2 weeks ago' +%Y-%m-%dT%H:%M:%SZ)" --format="value(name)")
 if [ -n "$images" ]; then
-	echo "$images" | xargs -I '{}' gcloud compute images delete '{}' --project="openshift-node-devel"
+    echo "The following images will be deleted:"
+    echo "$images"
```

Comment:
Same here.

Reply:
It's good to have that information while debugging.